internal/controller/controllers/agentserviceconfig_controller.go

```diff
@@ -799,6 +803,13 @@ func (r *AgentServiceConfigReconciler) newImageServiceDeployment(ctx context.Con
 		deployment.Spec.Template.Spec.Containers = []corev1.Container{container}
 		deployment.Spec.Template.Spec.Volumes = volumes
 		deployment.Spec.Template.Spec.ServiceAccountName = imageServiceName
+
+		nodeSelector := make(map[string]string)
```

Comment:
Is it possible for node selectors or tolerations to be manually modified by a user? 

said another way, do we really want to clobber the nodeSelector + tolerations on every reconcile?

Reply:
I don't think there's any expectation that manual changes can be applied to this Deployment.